core/src/main/scala/kafka/admin/ConfigCommand.scala

```diff
@@ -349,6 +368,15 @@ object ConfigCommand extends Config {
       .toSeq
   }
 
+  /**
+    * Returns all the valid broker logger configurations
+    */
+  private def brokerLoggerConfigs(adminClient: Admin, entityName: String): Seq[ConfigEntry] = {
+    val configResource = new ConfigResource(ConfigResource.Type.BROKER_LOGGER, entityName)
+    val configs = adminClient.describeConfigs(Collections.singleton(configResource)).all.get(30, TimeUnit.SECONDS)
```

Comment:
The previous code had a 30 second timeout for describe and a 60 second for alter, so I kept with the convention